### PR TITLE
Linux WPT tests on GitHub Action

### LINKED ISSUE #28280: Run WPT tests (linux)

### PR DESCRIPTION
**Pull request by pecastro**
- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #28280
- [X] These changes do not require tests because they add to an existing GitHub Action to run the Linux WPT tests.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: CI
 on:
   # Trigger CI when making pushes to try branches, or the branch that homu interacts with.
   push:
-    branches: [ "github-actions-dev", "auto", "try", "try-linux", "try-mac", "try-windows" ]
+    branches: [ "github-actions-dev", "auto", "try", "try-linux", "try-mac", "try-windows", "linux-wpt-tests" ]
```
Comment (jdm) [inline .github/workflows/main.yml:6]: Please add `try-wpt` as well.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -894,852 +894,734 @@ jobs:
...
+
 
   build_result:
     name: homu build finished
     runs-on: ubuntu-latest
-    needs: ["build-win", "build-mac", "build-linux"]
+    needs: ["build-linux", "linux-wpt-1", "linux-wpt-2", "linux-wpt-3", "linux-wpt-4", "linux-wpt-5", "linux-wpt-6", "linux-wpt-7", "linux-wpt-8", "linux-wpt-9", "linux-wpt-10", "linux-wpt-11", "linux-wpt-12", "linux-wpt-13", "linux-wpt-14", "linux-wpt-15", "linux-wpt-16", "linux-wpt-17", "linux-wpt-18", "linux-wpt-19", "linux-wpt-20"]
     steps:
+      - name: What do we have here
+        run: ls -altr
```
Comment (jdm) [inline .github/workflows/main.yml:1624]: We can get rid of this step, right?

File: etc/taskcluster/decision_task.py [hunk 3]
```diff
@@ -37,8 +37,8 @@ def tasks(task_for):
             windows_arm64,
             windows_uwp_x64,
             macos_unit,
-            linux_wpt,
-            linux_wpt_layout_2020,
+            # linux_wpt,
+            # linux_wpt_layout_2020,
```
Comment (jdm) [inline etc/taskcluster/decision_task.py:41]: We should make the `test-wpt` branch just below this an empty list, too.

**Review by jdm — CHANGES_REQUESTED** [review]
We need to make these changes as part of https://github.com/servo/servo/blob/master/etc/ci/workflow.mako and run etc/ci/generate_workflow.py, rather than directly in main.yml.

File: etc/taskcluster/decision_task.py (hunk 3 above)
Comment (jdm) [reply etc/taskcluster/decision_task.py:41]: Oh, I see - for simplicity's sake, should disable the macos_wpt tests as well, and empty the try-wpt and and try-wpt-mac branches below as well.

**Comment by jdm** [comment]
You will need to add `# pragma: no cover` lines like https://github.com/servo/servo/blob/d7b7292d1804e760c8e9c359fb1736ddfd50ca4b/etc/taskcluster/decision_task.py#L896 to the lines that are reported by:
```
etc/taskcluster/decision_task.py     188     40    79%   493, 512-515, 543-549, 561, 565, 569-572, 578-713
etc/taskcluster/decisionlib.py       328      1    99%   694
```

**Comment by pecastro** [comment]
Wouldn't these be better off commented or removed from the script if they're not being run rather than marked with pragma ?

**Comment by jdm** [comment]
True. Go ahead!

File: etc/taskcluster/decision_task.py [hunk 4]
```diff
@@ -60,9 +57,10 @@ def tasks(task_for):
             "try-linux": [linux_tidy_unit, linux_docs_check, linux_release],
             "try-windows": [windows_unit, windows_arm64, windows_uwp_x64],
             "try-arm": [windows_arm64],
-            "try-wpt": [linux_wpt],
+            "try-wpt": [],
             "try-wpt-2020": [linux_wpt_layout_2020],
```
Comment (jdm) [inline etc/taskcluster/decision_task.py:61]: `try-wpt-2020` also needs to be an empty list.

**Comment by pecastro** [comment]
@jdm Workflow needs approval to run ?

File: etc/taskcluster/decision_task.py (hunk 4 above)
Comment (pecastro) [reply etc/taskcluster/decision_task.py]: Done.

**Comment by pecastro** [comment]
Different failures against my branch https://github.com/pecastro/servo/runs/2461925580 might hint at new timeouts? I've pushed again to take it through another run.

**Comment by pecastro** [comment]
@jdm Another r+ please ?

**Comment by jdm** [comment]
It just means that the test results are intermittent in this CI setup.

@bors-servo r+

**Pull request merged**